packages/pharos/src/components/switch/pharos-switch.scss

```diff
@@ -0,0 +1,127 @@
+@use '../../utils/scss/mixins';
+
+.input-wrapper {
+  @include mixins.option-wrapper;
+
+  align-items: center;
+  column-gap: var(--pharos-spacing-1-x);
+}
+
+.switch__control {
+  height: 32px;
+  width: 70px;
```

Comment:
Are these something that can/should be calculated from other values, or are they pretty squarely hand-crafted? Same question really for any hard-coded values, though some needing more scrutiny than others I'm sure.

Reply:
From the mockup they appear to rather handcrafted. Technically 32px is 2rem but that could be mostly coincidental? @fswlee anything here you suggest or worth adding context to?

Reply:
the little knob is 1rem x 1rem and there's .5rem space above ad below which creates the height. the width is a little more arbitrary. we could consider being more specific and intentional with our values, for sure. let me look at some options

Reply:
Ahh i might have the knob 1px small in both dimensions right now at 15px. I can tweak as part of this too